data/stats/get_flight_stats.py

```diff
@@ -0,0 +1,34 @@
+"""
+    Gather flight statistics for each 
+    airport from given aiport flight data. 
+"""
+
+from dataclasses import dataclass, field
+import pandas as pd
+from src.data.types import Flight
+
+@dataclass
+class FlightStats:
+    flights: pd.DataFrame
+    flights_count_list: pd.DataFrame = field(init = False)
+    def __post_init__(self) -> None:
+        self.flights_count_list = self.flights.map(self.count_flights)
```

Comment:
I'm not sure I understand the need for this additional df?

Reply:
Are you talking about flights_count_list?

Reply:
Yes, is there an advantage to creating a separate attribute instead of modifying `self.flights` here?

Reply:
Hmmm not really I guess I can just add new columns to `self.flights`